### ISSUE TITLE
Ajax cross-domain requests does not set 'with credentials'

### ISSUE THREAD

**Issue by seanislegend** [Reactive-Extensions/RxJS-DOM#95]
Creating an Ajax request with the option ```crossDomain``` set to true does not set ```withCredentials```. 
Furthermore, there is no option to manually change this option. 

Is this expected? I have overcome this issue by updating the source and adding a new line in ```getCORSRequest()```:
```
if ('withCredentials' in xhr) {
  return xhr;
}
```
is now:
```
if ('withCredentials' in xhr) {
  xhr.withCredentials = true;
  return xhr;
}
```

Any help/advice would be appreciated.

**Comment by mattpodwysocki** [comment]
@seanislegend yes, that is a bug on our part and is fixed as per 7.0.3

**Comment by jaredatron** [comment]
I'm seeing the same bug in `7.0.3`
